proposals/sdr3apisWC.md

```diff
@@ -0,0 +1,58 @@
+# SDR3 Ingest Pipeline / DataDevops SDR3 Work Cycle
+
+*(proposal being written for possible start later this fall / early this winter)*
+
+There are three converging areas of need in this proposal:
+
+1. having a devOps & dataOps work cycle to continue to build out analysis and work around data API-forward architecture for DLSS, getting production-ready / operationalizing our AWS usage, and consolidating analysis on the current state and desired shifts of our existing infrastructure around SDR.
+2. pulling together and push forward analysis of future SDR system requirements outside of the Hydrox demo, then matching that with design and at least one major example of a portion of SDR architecture and infrastructure migrated to a data-forward, API-reliant and cloud-based codebase.
+3. starting the reset of the Google Books accessioning pipeline, which requires a full rewrite of its current ingest and accessioning processes into SDR. Due to its size (over 2 million digital books) and current backlog status, the Google books accessioning pipeline is an ideal candidate for a test of purpose and design of the first two work areas.
+
+## Goals:
+
+- kick start "SDR3" work by
+  - being a parallel effort to Hydrox, which needs this work to understand interfaces between current & future SDR to Hydrox
+  - begin to define the requirements & business cases for components of SDR and their future form
+  - offer more details & options to the emerging Chimera reboot (digital library review) and data architecture meeting(s)
+- generate and share understanding on our current systems (including ability to work with components for a phased replacement plan)
+  - interview infrastructure practitioners on current pain points on current infrastructure (a "SDR2" retrospective)
+  - compile list of lessons learned
+  - reverse engineer specs and requirement lists for current digital library system needs (based on interviews and Current State work)
+- explore a data oriented, event driven architecture to process some data object that we currently accession via the SDR ‘pipeline’
+  - build a Google books ingest pipeline to vet the said approach
+  - generate a set of conceptual diagrams and documentation of concerns approached
+  - generate a wireframe of a strategic plan for migration to said approach
+
+## User Stories / Business Value(s):
+
+- As an Hydrox engineer, I would like to know what services (current or ideal) need to be connected to from the Hydrox / Fedora 3 implementations in order to fulfill the rest of the SDR functionalities (preservation, SearchWorks discovery, availability in an advanced administration interface, among others).
```

Comment:
I think this would be clearer written as:
> I would like to know what services the Hydrox implementation can leverage *OR* what services are needed and are not present in the current architecture.